packages/api/test/__fixtures__/sdk/alby/index.ts

```diff
@@ -0,0 +1,713 @@
+import type * as types from './types';
```

Comment:
Added this new test fixture to test out the awful issues I uncovered with discriminators. I'd rather have that case explicitly tested here than hidden in the smoketest.